### ISSUE TITLE
.sh miss the header

### ISSUE THREAD

**Issue by fabienwang** [deavmi/Picy#22]
#!/bin/sh
or 
#!/bin/bash

I suggest the first one as it can go POSIX and can be compatible with different shells.

**Comment by deavmi** [comment]
Yeah will get to that stuff.

I love how bash can simply read the file `./foo.sh` check for the `#!` and send the remaining file contants to that program. And then that program will interpret the file.

**Comment by deavmi** [comment]
@fabienwang Good point, but I'm a bash boy. So I will be going with the second executable, that being `bash` located at `/bin/bash`.

**Comment by deavmi** [comment]
@fabienwang Thanks BTW. :)

**Comment by fabienwang** [comment]
@deavmi NP Buddy

**Comment by deavmi** [comment]
@fabienwang :wink:
